### DISCUSSION TITLE
z offset saving in v1.3.2 - it's performed elsewhere.

### DISCUSSION THREAD

**Discussion by orangoo** [Jyers/Marlin#681] [General Discussion]
Hi

Whilst a print occurs we can adjust the z offset value in the Tune menu.

Previously once that value is adjusted during a live print it is stored in the menu and returns after a power cycle.

Now what happens is you can still adjust the z offset value in the Tune menu but that value isn't stored after a power cycle. What needs to happen is to head to 
1. Prepare 
2. Z Offset 
3. select the Z offset row and adjust that to the value you had when the z offset value was adjusted during a live print.
4. Save

When I power cycle I can see the Z offset value is to the value saved from the steps above

Mentioning this as I couldn't figure out why my z offset value wasn't saving but it does when using the 4 steps above.

Hope this helps everyone

**Comment by Jyers** [comment]
The Z offset was never saved after adjusting from the tune menu, perhaps you saved after your print or had a save command in your gcode somewhere
